examples/index.jsx

```diff
@@ -63,96 +62,99 @@ let CONSTANT_EXAMPLE;
     updateLang();
   });
 */
-const destroy = i18n.onUpdate('common', function () {
-  CONSTANT_EXAMPLE = i18n.renderI18n('common.header', {
-    project: 'i18n',
-  });
-});
 
 // Create Example Component
-const Examples = React.createClass({
-  getInitialState: function() {
-    return {
-      count: 0,
-    }
-  },
-  clickHandler: function () {
+class Example extends Component {
+  state = {
```

Comment:
This doesn't have to happen in the constructor anymore?

Reply:
[Nope!](https://babeljs.io/blog/2015/06/07/react-on-es6-plus#property-initializers)

Reply:
Neato!